clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala

```diff
@@ -130,6 +132,49 @@ object Exporter{
   final val defaultParallelism = 10
 }
 
+object Main {
+  def main(args: Array[String]) {
+    val conf = new Conf(args)
+
+    val spark = SparkSession.builder().getOrCreate()
+    val sc = spark.sparkContext
+
+    val endpoint = sc.hadoopConfiguration.get("lakefs.api.url")
+    val accessKey = sc.hadoopConfiguration.get("lakefs.api.access_key")
+    val secretKey = sc.hadoopConfiguration.get("lakefs.api.secret_key")
+
+    val apiClient = new ApiClient(endpoint, accessKey, secretKey)
+    val exporter = new Exporter(spark, apiClient, conf.repo(), conf.rootLocation())
+
+    if (conf.commit_id.isSupplied) {
+      exporter.exportAllFromCommit(conf.commit_id())
+      return
+    }
+
+    if (conf.prev_commit_id.isSupplied) {
+      exporter.exportFrom(conf.branch(), conf.prev_commit_id())
+      return
+    }
+
+    exporter.exportAllFromBranch(conf.branch())
+  }
+}
+
+
+/** Conf options and arguments usage is documented in https://github.com/scallop/scallop/wiki
+ *
+ */
+class Conf(arguments: Seq[String]) extends ScallopConf(arguments) {
+  val branch = opt[String](required=false)
+  val commit_id = opt[String](required=false)
+  val prev_commit_id = opt[String](required=false)
```

Comment:
According to https://github.com/scallop/scallop/issues/150#issuecomment-336894579, scallop converts camelCase to dash-case. So you can use `commitID`, `prevCommitID` etc.